src/app/private/scripts/controllers/announcement/createAnnouncementController.js

```diff
@@ -1,65 +1,43 @@
 'use strict'
-angular.module('playerApp').controller('createAnnouncementCtrl', ['$rootScope', '$scope', '$timeout', 'config', 'toasterService', 'announcementService', 'fileUpload',
-  function ($rootScope, $scope, $timeout, config, toasterService, announcementService, fileUpload) {
-    var createAnn = this
-    createAnn.data = {}
-    createAnn.attachment = []
-    createAnn.senderlist = []
-    createAnn.targetIds = []
-    createAnn.disableBtn = true
-    createAnn.showUrlField = false
-    createAnn.errorFlag = false
-    createAnn.stepNumber = 1
-    createAnn.isMetaModified = false
-    createAnn.announcementType = []
-    createAnn.repeatableWebLinks = []
-    createAnn.selectedRecipients = []
-    createAnn.hideAnncmntBtn = false
-    createAnn.uploadAttchement = false
-
-    var getDefinitionReq = {
-      request: {
-        'rootorgid': $rootScope.rootOrgId,
-        'userid': $rootScope.userId,
-        'definitions': ['announcementtypes', 'senderlist']
+angular.module('playerApp').controller('createAnnouncementCtrl', ['$rootScope', '$scope', '$state', '$stateParams', '$timeout', 'config', 'toasterService', 'announcementService', 'fileUpload', 'AnnouncementModel', 'announcementAdapter', 'portalTelemetryService',
+  function ($rootScope, $scope, $state, $stateParams, $timeout, config, toasterService, announcementService, fileUpload, AnnouncementModel, announcementAdapter, portalTelemetryService) {
+    var composeAnn = this
+    composeAnn.data = {}
+    composeAnn.attachment = []
```

Comment:
these fields should not be required. they are provided in the announcement model.